src/Microsoft.Security.Utilities.Core/WellKnownRegexPatterns.cs

```diff
@@ -17,95 +19,66 @@ public static class WellKnownRegexPatterns
-    {
-        new UnclassifiedJwt(),
-        new UrlCredentials(),
-        new LooseSasSecret(),
-        new OAuth2BearerToken(),
-        new Unclassified32ByteBase64String(),
-        new Unclassified64ByteBase64String(),
-        new AadClientAppLegacyCredentials34(),      // SEC101/101 legacy generated passwords.
-        new Pkcs12CertificatePrivateKeyBundle(),
-        new Unclassified16ByteHexadecimalString(),
-    };
-
-    public static IEnumerable<RegexPattern> HighConfidenceMicrosoftSecurityModels { get; } = new RegexPattern[]
-    {
-        new UnclassifiedLegacyCommonAnnotatedSecurityKey(),
-        new AadClientAppIdentifiableCredentials(),
-        new AzureFunctionIdentifiableKey(),
-        new AzureSearchIdentifiableQueryKey(),
-        new AzureSearchIdentifiableAdminKey(),
-        new AzureRelayIdentifiableKey(),
-        new AzureEventHubIdentifiableKey(),
-        new AzureServiceBusIdentifiableKey(),
-        new AzureIotHubIdentifiableKey(),
-        new AzureIotDeviceIdentifiableKey(),
-        new AzureIotDeviceProvisioningIdentifiableKey(),
-        new AzureStorageAccountIdentifiableKey(),
-        new AzureCosmosDBIdentifiableKey(),
-        new AzureBatchIdentifiableKey(),
-        new AzureMLWebServiceClassicIdentifiableKey(),
-        new AzureApimIdentifiableDirectManagementKey(),
-        new AzureApimIdentifiableSubscriptionKey(),
-        new AzureApimIdentifiableGatewayKey(),
-        new AzureApimIdentifiableRepositoryKey(),
-        new AzureCacheForRedisIdentifiableKey(),
-        new AzureContainerRegistryIdentifiableKey(),
-        new NuGetApiKey(),
-        new AdoLegacyPat(),                         // SEC101/102
-        new AzureCosmosDBLegacyCredentials(),       // SEC101/104
-        new AzureStorageAccountLegacyCredentials(), // SEC101/106
-        new AzureMessagingLegacyCredentials(),
-        new AzureDatabricksPat(),
-        new AzureEventGridIdentifiableKey(),
-    };
-
-    public static IEnumerable<RegexPattern> DataClassification { get; } = new RegexPattern[]
-    {
-        new IPv4(),
-        new IPv6(),
-        new Float(),
-        new Integer(),
-        new GuidValue(),
-    };
-
-    public static IEnumerable<RegexPattern> HighConfidenceThirdPartySecurityModels { get; } = new List<RegexPattern>
-    {
-        new NpmAuthorKey(),
-        new SecretScanningSampleToken(),
-    };
+    public static IReadOnlyList<RegexPattern> UnclassifiedPotentialSecurityKeys { get; } =
+        CreateList(new UnclassifiedJwt(),
+                   new UrlCredentials(),
+                   new LooseSasSecret(),
+                   new OAuth2BearerToken(),
+                   new Unclassified32ByteBase64String(),
+                   new Unclassified64ByteBase64String(),
+                   new AadClientAppLegacyCredentials34(),      // SEC101/101 legacy generated passwords.
+                   new Pkcs12CertificatePrivateKeyBundle(),
```

Comment:
>Pkcs12CertificatePrivateKeyBundle [](http://example.com/codeflow?start=23&length=33)

strictly speaking, this is a classified secret. the issue here is one of confidence level. btw - looking at the regex, we could certainly improve accuracy.

this rule is also interesting to discuss because we could definitely drive our confidence rate to a very high degree if we're willing to attempt deserializing this data. i'm assuming this should be possible without pulling in unwanted deps like bouncycastle, etc., (i.e., with built-in .net crypto support).

I opened a bug on this. The question to you is whether you want this precisely classified (but medium confidence thing) in this collection.